src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --another-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  position: relative;
+}
+
+.header__logo-link {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.header__nav-list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  margin-left: 20px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). In this case, you've added a left margin to the `.nav__link` class. It's better to add margins to only one side (either left or right) to avoid unexpected behavior.